docs/area-owners.md

```diff
@@ -134,6 +134,7 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
 | area-System.Xml                                | @jeffhandley  | @dotnet/area-system-xml                             |                                                                                                                                                                                                                                                                                           |
 | area-Threading-mono                            | @SamMonoRT    | @lambdageek                                         |                                                                                                                                                                                                                                                                                           |
 | area-TieredCompilation-coreclr                 | @mangod9      | @kouvel                                             |                                                                                                                                                                                                                                                                                           |
+| area-Tools-ILLink                | @agocke      | @sbomer                                             |                                                                                                                                                                                                                                                                                           |
```

Comment:
Could you introduce @dotnet/area-illink alias, it would help to tag all interested folks ad hoc as well